docs/xks/developer-guide/container-security.md

```diff
@@ -359,3 +359,9 @@ Report:
 ```
 
 The presence of a vulnerability does not mean that a patched version exists yet, or even that the image is inherently insecure. The vulnerability may affect components that are not used or have low risk to being exposed. It is however important to be conscious of the vulnerabilities that exist and consider future work to take responsibility for fixing them. Refer to the [official Starboard documentation](https://aquasecurity.github.io/starboard/latest/faq) for more specific documentation.
+
+## Pod max PID
+
+To make sure that a single pod can't take down a whole server aks noisy neighbours, we have set a max limit on how many PID:s a single pod can utilize.
```

Comment:
"To make sure that a single pod can't take down a whole server aks noisy neighbours". Sentence does not make sense to me